src/constraints/LessThanConstraint.jl

```diff
@@ -64,6 +61,10 @@ Base.:<=(lhs::AbstractExpr, rhs::Value) = <=(lhs, constant(rhs))
 Base.:<=(lhs::Value, rhs::AbstractExpr) = <=(constant(lhs), rhs)
 
 function populate_dual!(model::MOI.ModelLike, c::LessThanConstraint, indices)
+    # FIXME(odow): this dual is the 'wrong' sign, because Convex implements
+    # rhs - lhs in Nonnegatives for a LessThanConstraint, instead of
+    # lhs - rhs in Nonpositives.
+    # Should we fix this?
     ret = MOI.get(model, MOI.ConstraintDual(), indices)
     c.dual = output(reshape(ret, c.size))
```

Comment:
So it's horribly breaking to change the sign of this dual. But this is an annoying inconsistency.